util/cron/common-slurm-gasnet.bash

```diff
@@ -1,26 +1,27 @@
 #
 # Configure environment for
-# multi-node, multi-locale testing on a linux64 cluster with slurm-gasnetrun_ibv launcher.
+# multi-node, multi-locale testing on a cray-cs with slurm-gasnetrun_*
+# launcher.
 
-# This should be sourced by other scripts that wish to make use of the variables set here.
+# This should be sourced by other scripts that wish to make use of the variables
+# set here.
 
 CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
 
 source $CWD/common.bash
+source $CWD/common-cray-cs.bash y
 
 export CHPL_COMM=gasnet
-export CHPL_COMM_SUBSTRATE=ibv
-export CHPL_LAUNCHER=slurm-gasnetrun_ibv
-
-export CHPL_LAUNCHER_USE_SBATCH=1
-export CHPL_APP_LAUNCH_CMD=$CHPL_HOME/util/test/chpl_launchcmd.py
 
 unset CHPL_START_TEST_ARGS
 nightly_args="${nightly_args} -no-buildcheck"
 
+# we use at most 8 locales in examples and hellos
+export CHPL_TEST_NUM_LOCALES_AVAILABLE=8
```

Comment:
Why set `CHPL_TEST_NUM_LOCALES_AVAILABLE`?

Reply:
Out of caution. Before, it was set to the number of nodes allocated, and it is only used for skipping tests that have larger numlocales than this (and it was causing 3 chplvis tests to be skipped). I can drop it. It probably makes more sense on systems with more limited resources.

Reply:
Yeah, I think we can drop it.